lib/firmata.js

```diff
@@ -480,8 +504,13 @@ function Board(port, options, callback) {
     TYPE: {
       DRIVER: 1,
       TWO_WIRE: 2,
+      THREE_WIRE: 3,
       FOUR_WIRE: 4,
     },
+    STEPTYPE: {
```

Comment:
This name might be confusing with the unrelated `TYPE` property? Can we call it `STEP_SIZE`?